### ISSUE TITLE
Always spinning progress in GraphiQL IDE

### ISSUE THREAD

**Issue by odrowonz** [wp-graphql/wp-graphql#2881]
### Description

I am using WordPress 6.2.2 and PHP FastCGI (Apache) 8.2.0 (alt). After installing [release v1.14.9](https://github.com/wp-graphql/wp-graphql/releases/download/v1.14.9/wp-graphql.zip) and activating the plugin, I go to the GraphiQL IDE and see an endless process of progress rotation. The situation is similar when the plugin version is downgraded to 1.6.12 (I went through all the versions). Version 1.6.12 successfully opened the GraphiQL IDE for me. What could be wrong?
![ErrorOfGraphQL](https://github.com/wp-graphql/wp-graphql/assets/16529784/21cea775-8cbc-47ae-a7c7-d50553f78474)

### Steps to reproduce

1. Go to Plugins and install and activate [release v1.14.9](https://github.com/wp-graphql/wp-graphql/releases/download/v1.14.9/wp-graphql.zip).
2. Go to GraphiQL IDE.
3. See the always spinning progress.
![ErrorOfGraphQL](https://github.com/wp-graphql/wp-graphql/assets/16529784/30076f48-db3e-46db-91b2-80ac35cad2e2)

### WPGraphQL Version

1.14.9

### WordPress Version

6.2.2

### PHP Version

FastCGI (Apache) 8.2.0 (alt)

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for WPGraphQL.

- [X] Yes
- [X] My issue is with compatibility with a specific WordPress plugin, and I have listed all my installed plugins (and version info) above.

**Comment by justlevine** [comment]
Hey @odrowonz 

Please follow the [Debugging steps](https://www.wpgraphql.com/docs/debugging). After you follow those steps, if you take a look at the [Network Response](https://www.wpgraphql.com/docs/debugging#checking-the-response), you should see a more explicit error. Once you share that we'll have a better chance of being able to help.

(Note: this plugin [isn't actively tested against PHP 8.2 yet](https://github.com/wp-graphql/wp-graphql/blob/027323f9f756ae7045194dcf5f2754d3f5533e9d/.github/workflows/testing-integration.yml#L26), so that could be a contributing factor).

**Comment by roenfeldt** [comment]
I second @odrowonz 's problem, the same is happening to my WordPress 6.3 installation + WPGraphQL 1.14.10. 

The issue doesn't seem to be related to PHP 8.2. I tried changing the PHP version in my hosting cPanel to 8.1, 8.0, and even 7.4, but that doesn't make any difference. 

The four-dots spinner won't go away, and the browser's Inspection Tools are showing a JavaScript-related error. Hopefully the attached image below will help you @jasonbahl figure out the issue. 

Thank you!
![wpgraphql](https://github.com/wp-graphql/wp-graphql/assets/60813178/40b0f241-fed1-4ecb-b27a-08d5b34c4ff9)

**Comment by justlevine** [comment]
Hey @roenfeldt 👋

It's likely not the _same_ bug (the IDE not loading is just a sign that there is _a_ bug/conflict), but since @odrowonz doesn't seem to be using this ticket, I'm happy to help you debug this here in the off chance it does help the OP 😎

There's one thing missing from your screenshot that I'd need to begin: the `Network Response` (in your screenshot, it's on the top right of the console, the tab next to the `Headers`. Please make sure to turn on GraphQL debugging first 🙏).
That will show us what the server is trying to send to the GraphiQL IDE instead of the expected introspection.

**Comment by jasonbahl** [comment]
@odrowonz I attempted to reproduce on PHP 8.2 (we don't currently have automated tests for 8.2, so I thought maybe there's a chance that was the culprit) but I'm able to load GraphiQL IDE just fine on PHP 8.2

Here's the server details of the local server I set up: 

![CleanShot 2023-08-09 at 11 43 38](https://github.com/wp-graphql/wp-graphql/assets/1260765/e5029a05-ebd6-4aef-9909-ab0b1e586a02)

And GraphiQL IDE loads fine: 

![CleanShot 2023-08-09 at 11 44 27](https://github.com/wp-graphql/wp-graphql/assets/1260765/ed3f2080-9f07-4c58-800f-3bc18dfe3ec6)

There's a good chance there could be an issue with a plugin or theme you have active, so be sure to test in a clean install vs an install with many other existing plugins. 

I do see that you have at least Jetpack and SpeedyCache active in your screenshot. It's important for us to know if we're dealing with an issue with WPGraphQL itself, a plugin conflict with a specific plugin / combination of plugins, or something else. 

I also notice you might have WordPress set to use a different language, so there's a chance that's impacting this 🤔

**Comment by jasonbahl** [comment]
@odrowonz it looks like you're running WordPress in non-English, and if you have a custom theme active that has page templates that use non-ascii characters in the template name, there's a good chance that you're running into this issue: https://github.com/wp-graphql/wp-graphql/issues/2807

**Comment by jasonbahl** [comment]
@roenfeldt are you using non-ascii characters in page templates? Curious if you might also be running into https://github.com/wp-graphql/wp-graphql/issues/2807

**Comment by odrowonz** [comment]
Yes, I use default theme "Twenty Twenty-Three" and pages with Russian encoding.

**Comment by jasonbahl** [comment]
@odrowonz what about page templates? Like page-*.php files with Russian encoding? 

If it's just page titles, that won't impact GraphiQL but page templates will.

**Comment by odrowonz** [comment]
I did an experiment. I installed standard Wordpress, and chose the English scheme, which is "Twenty Twenty-Three" by default. Then I switch the language to Russian in settings -> general. Then GraphiQL turns the hourglass - is not working. I return the language back to English. Then GraphiQL starts working . And of course all *.php files have English encoding.

**Comment by jasonbahl** [comment]
@odrowonz ok nice. This is good information. Will try to reproduce!

**Comment by justlevine** [comment]
@jasonbahl @odrowonz it sounds like #2492  -  changing the local translates the WP template name (#2807 is just a manual subset of the same issue).

**Comment by odrowonz** [comment]
Colleagues, is there any deadline for fixing this bug?

**Comment by justlevine** [comment]
@odrowonz assuming your issue is the same as the [ones I linked previously](https://github.com/wp-graphql/wp-graphql/issues/2881#issuecomment-1675924412) (and if so, please let us know so we can consolidate the tickets 🙏), then you can try #2808 and see if it resolves your issue.

That particular PR needs feedback and unit testing before it can be merged, and if it doesn't solve things for you then we likely need a separate fix. Either way, contributions are always welcome!

**Comment by odrowonz** [comment]
> @odrowonz assuming your issue is the same as the [ones I linked previously](https://github.com/wp-graphql/wp-graphql/issues/2881#issuecomment-1675924412) (and if so, please let us know so we can consolidate the tickets 🙏), then you can try #2808 and see if it resolves your issue.
> 
> That particular PR needs feedback and unit testing before it can be merged, and if it doesn't solve things for you then we likely need a separate fix. Either way, contributions are always welcome!

Is there any release that includes this edit? (I tried 1.14.10 and it doesn't work there yet)

**Comment by justlevine** [comment]
@odrowonz 

> assuming your issue is the same as the https://github.com/wp-graphql/wp-graphql/issues/2881#issuecomment-1675924412 (and if so, please let us know so we can consolidate the tickets 🙏),

You seemed to miss ☝️ part of my last reply, but from your responses I'm going to assume that you read that issue and it _is_ a duplicate of this one. As such, I'm closing this in favor of #2492 (if I'm wrong, let us know and we'll reopen).

> Is there any release that includes this edit? (I tried 1.14.10 and it doesn't work there yet)

Like I [said](https://github.com/wp-graphql/wp-graphql/issues/2881#issuecomment-1678604680), that particular PR needs feedback and unit testing before it can be merged. 

> Can you compile pre-release for test?

To "compile" that or any branch, you can pull / download it locally, and then run `composer install --no-dev --optimize-autoloader` from inside the plugin folder. (or just `composer install` if you want to include the dev-deps needed to contribute).
